### PR TITLE
Adding assertions for AddChange test scenario for ResultV2 Test

### PR DESCRIPTION
**Pull request by zhaque44**
The updated test accounts for much needed assertions for AddChange scenario in Result V2 test:

This code snippet initializes an **expectedFileChanges variable**, which is containing the expected changes. Each file is associated with a set of object changes represented by ObjectChanges. Within each file's changes, specific objects are associated with an array of Change structs, where each struct represents a change in values. The g.Expect(result.FileChanges) statement asserts that the FileChanges field of the result variable matches the expected changes defined.

### CODE REVIEW

**Comment by zhaque44** [comment]
Hi @souleb hope you are doing well, I made the necessary changes recommended here: https://github.com/fluxcd/image-automation-controller/pull/682#issuecomment-2122409405

**Comment by zhaque44** [comment]
@souleb hey can you take a look at these changes  please

File: pkg/update/result_test.go [hunk 1]
```diff
@@ -117,6 +117,28 @@ func TestResultV2(t *testing.T) {
...
+			objectNames[1]: []Change{
+				{
+					OldValue: "cccc:v1.0",
+					NewValue: "cccc:v1.2",
+					Setter:   "foo-ns:policy",
+				},
+			},
+		},
+	}
+	g.Expect(result.FileChanges).To(Equal(expectedFileChanges))
```
Comment (stefanprodan) [inline pkg/update/result_test.go:140]: This is a duplicate of the test right below it.

Comment (zhaque44) [reply pkg/update/result_test.go:140]: Hi @stefanprodan  thanks for pointing that out I’ll close this PR

**Pull request closed without merging**